core/src/components/select-popover/test/basic/select-popover.e2e.ts

```diff
@@ -0,0 +1,63 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+import type { SelectPopoverOption } from '../../select-popover-interface';
+import { SelectPopoverPage } from '../fixtures';
+
+test.describe('select-popover: basic', () => {
+  test.beforeEach(({ skip, browserName }) => {
+    skip.rtl();
+    skip.mode('ios', 'Consistent behavior across modes');
+    test.skip(browserName === 'webkit', 'https://ionic-cloud.atlassian.net/browse/FW-2979');
+  });
+
+  const options: SelectPopoverOption[] = [
+    { value: 'apple', text: 'Apple', disabled: false, checked: false },
+    { value: 'banana', text: 'Banana', disabled: false, checked: false },
+  ];
+
+  test.describe('single selection', () => {
+    let selectPopoverPage: SelectPopoverPage;
+
+    test.beforeEach(async ({ page }) => {
+      selectPopoverPage = new SelectPopoverPage(page);
+      await selectPopoverPage.setup(options, false);
+    });
+
+    test('clicking an unselected option should dismiss the popover', async () => {
+      await selectPopoverPage.clickOption('apple');
+      await selectPopoverPage.ionPopoverDidDismiss.next();
+      await expect(selectPopoverPage.popover).not.toBeVisible();
+    });
+
+    test('clicking a selected option should dismiss the popover', async () => {
+      await selectPopoverPage.updateOptions([
+        { value: 'apple', text: 'Apple', disabled: false, checked: true },
+        { value: 'banana', text: 'Banana', disabled: false, checked: false },
+      ]);
+
+      await selectPopoverPage.clickOption('apple');
+      await selectPopoverPage.ionPopoverDidDismiss.next();
+      await expect(selectPopoverPage.popover).not.toBeVisible();
+    });
+
+    test('pressing Space on an unselected option should dismiss the popover', async () => {
+      await selectPopoverPage.pressSpaceOnOption('apple');
+      await selectPopoverPage.ionPopoverDidDismiss.next();
+      await expect(selectPopoverPage.popover).not.toBeVisible();
+    });
+
+    test('pressing Space on a selected option should dismiss the popover', async ({ browserName }) => {
+      test.skip(browserName === 'firefox', 'Same behavior as https://ionic-cloud.atlassian.net/browse/FW-2979');
+
+      await selectPopoverPage.updateOptions([
+        { value: 'apple', text: 'Apple', disabled: false, checked: true },
+        { value: 'banana', text: 'Banana', disabled: false, checked: false },
+      ]);
```

Comment:
Is there a way we can start the test with these options pre-configured in the `<script>` tag? One thing that is slowing our tests down on CI is the amount of "setup work" we do before we begin our tests. This operation likely runs pretty quickly on your computer, but it can take ~1-2 seconds on CI (which adds up pretty quickly if many tests are doing this)